prime-router/src/test/kotlin/fhirengine/engine/FhirTranslatorTests.kt

```diff
@@ -172,12 +230,21 @@ class FhirTranslatorTests {
         mockkObject(BlobAccess)
 
         // set up
-        val settings = FileSettings().loadOrganizations(colorado)
```

Comment:
Annoyingly a bunch of these tests weren't really doing anything.  None of the receivers in Colorado org match the receiver in the FHIR bundle.